recipes/root/all/conanfile.py

```diff
@@ -56,6 +56,9 @@ class RootConan(ConanFile):
         # "libuuid/1.0.3",
     )
 
+    def configure(self):
+        tools.check_min_cppstd(self, "11")
```

Comment:
Yickys! you found a shortcoming in CCI, it currently does not handle `cppstd` and this should follow something like https://github.com/conan-io/conan/issues/8002 in order to have the most compatibility

you should be able to just copy paste 🤞 and tweak the compiler versions https://github.com/conan-io/conan-center-index/blob/689e2663fe41689259f61a5dbd3fe245e24c03b9/recipes/cxxopts/all/conanfile.py#L27

Reply:
Thanks. This is implemented in 7f503ea.